src/components/AddItems.jsx

```diff
@@ -28,6 +28,10 @@ export function AddItems({ data }) {
 				event.target.elements['purchase-date'].value;
 
 			try {
+				if (itemName === '') {
+					alert('Please add an item name.');
+					return;
+				}
```

Comment:
for some reason this check still allows for empty spaces to be added, i propose to add some regex to the input:
`<input  pattern="^[^\s].+[^\s]$"  />`

Breakdown of the expression:
- ^[^\s]: Asserts that the string starts with a character that is not whitespace.
- .+: Ensures that there is at least one or more characters between the start and end.
- [^\s]$: Asserts that the string ends with a character that is not whitespace.